src/main/java/reposense/git/GitCommitEditer.java

```diff
@@ -0,0 +1,19 @@
+package reposense.git;
+
+import reposense.model.RepoConfiguration;
+import reposense.system.CommandRunner;
+
+/**
+ * Executes git commands that will edit the repository's commit information.
+ */
+public class GitCommitEditer {
+
+    /**
+     * Removes all the author names for the commits that we are ignoring in {@code config}.
+     * Assumes that the list of commit hashes in {@code config} is given in reverse-chronological order.
```

Comment:
Why the assumption when this isn't compulsory as written in docs?

Reply:
Hmm, so we should rephrase it to be the same as in the docs: `For accurate results, the commits should be provided with their full hash in reverse chronological order.`?